### DISCUSSION TITLE
Switch message quantity indicator while scrolling through function options

### DISCUSSION THREAD

**Discussion by silverfrail** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#27] [Feature Requests & Ideas]
While editing a switch and scrolling through each function option (toggle on, toggle off, press, etc.), it would be nice to see a number to the side of each option that indicates how many messages are already assigned to that function.  This way you could more quickly and easily see if there is anything assigned to a switch that you may have forgotten about.

**Comment by simonaglover95** [comment]
This is visualised next to each message stack in the Web Editor 👍🏻
